kubernetes/windows/Dockerfile

```diff
@@ -30,7 +32,7 @@ COPY ./omsagentwindows/installer/conf/out_oms.conf /etc/omsagentwindows
 # copy keepcert alive ruby scripts
 COPY ./omsagentwindows/installer/scripts/rubyKeepCertificateAlive/*.rb /etc/fluent/plugin/
 
-ENV AGENT_VERSION "win-ciprod05262020-2"
+ENV AGENT_VERSION ${IMAGE_TAG}

```

Comment:
is this in readme to set IMAGE_TAG env before build ? I couldn't find it...

Reply:
> is this in readme to set IMAGE_TAG env before build ? I couldn't find it...

> yes, this being covered as note in the readme https://github.com/microsoft/Docker-Provider/tree/gangams/release-pipeline#build-cert-generator-out-oms-plugun-and-docker-image-and-publish-docker-image.  For prod release, release pipeline will push retag the images with tags our convention for both windows and linux.

Reply:
> > is this in readme to set IMAGE_TAG env before build ? I couldn't find it...
> 
> > yes, this being covered as note in the readme https://github.com/microsoft/Docker-Provider/tree/gangams/release-pipeline#build-cert-generator-out-oms-plugun-and-docker-image-and-publish-docker-image.  For prod release, release pipeline will push retag the images with tags our convention for both windows and linux.

> added win- prefix in the docker file so that same image name can be passed to build command.